primitives/sr25519/keys_test.go

```diff
@@ -197,6 +198,36 @@ func TestSecretKey(t *testing.T) {
 			t.Fatalf("sk == sk2")
 		}
 	})
+	t.Run("Ed25519Bytes", func(t *testing.T) {
+		ed25519Bytes, _ := hex.DecodeString("28b0ae221c6bb06856b287f60d7ea0d98552ea5a16db16956849aa371db3eb51fd190cce74df356432b410bd64682309d6dedb27c76845daf388557cbac3ca34")
+		sk, err := NewSecretKeyFromEd25519Bytes(ed25519Bytes)
+		if err != nil {
+			t.Fatalf("NewSecretKeyFromEd25519Bytes: %v", err)
+		}
+
+		pkBytes, err := sk.PublicKey().MarshalBinary()
```

Comment:
We can remove this, as we are testing if `PublicKey` method works. Here, it is enough to test that the secret key `sk.key` is ok, as public key is derived from it.